docs/docs/getting-started/cloud-providers.mdx

```diff
@@ -1,15 +1,32 @@
 ---
-title: Choosing cloud providers
+title: Supported Cloud Providers
```

Comment:
Note here @viniciusdc  since we are only listing the providers and not giving folks a detailed guide or else to choose between one or another I think this title is better